src/components/Sidebar/TrackKeys/TrackKey.jsx

```diff
@@ -11,7 +11,7 @@ const TrackKey = ({ track, toggleOpen, clickTrackButton }) => {
     hasButton,
     sideComponent
   } = track
-  const isExpandable = isOpen !== undefined
+  const isExpandable = (isOpen !== undefined) && tracks && (tracks !== undefined) && tracks.length > 0
```

Comment:
I'm not sure you can rely on this as `tracks` can be dynamically loaded i.e. only loaded in when you hit `toggleOpen` (could be wrong - been a while since I looked at this code)